Automattic/pocket-casts-android
# End of Year: Show total listening time for the year

| 📘 Project: #410 |
|:---:|

iOS PR: https://github.com/Automattic/pocket-casts-ios/pull/411

## Description

Show approximate total listening time in 2022. (Design not covered)

## Testing Instructions

1. Set `END_OF_YEAR_ENABLED` feature flag to `true` in `base.gradle`
1. Login to an existing account
2. Run the app
3. When the prompt appears, tap "View My 2022"
4. ✅ Check that your total listening time in 2022 is shown

## Screenshots or Screencast 
<img width=320 src="https://user-images.githubusercontent.com/1405144/199226374-c24bbe6d-b4bd-4765-acfe-adbfbff82926.png"/>

## Checklist

N/A
-  If this is a user-facing change, I have added an entry in CHANGELOG.md
- I have considered whether it makes sense to add tests for my changes
- All strings that need to be localized are in `modules/services/localization/src/main/res/values/strings.xml`
- Any jetpack compose components I added or changed are covered by compose previews
 
#### I have tested any UI changes...

- with different themes
- with a landscape orientation
- with the device set to have a large display and font size
- for accessibility with TalkBack
